<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="user-PC" ignored="0" name="Application Test" tests="4" failures="2" timestamp="24 Apr 2019 06:55:19 GMT" time="0.095" errors="0">
  <testcase name="doLogin" time="0.015" classname="testcases.ApplicationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	 Error message ">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
 Error message 
at testcases.ApplicationTest.doLogin(ApplicationTest.java:52)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="doPasswordChange" time="0.0" classname="testcases.ApplicationTest">
    <skipped/>
  </testcase> <!-- doPasswordChange -->
  <testcase name="logOut" time="0.0" classname="testcases.ApplicationTest"/>
  <testcase name="forgotPassword" time="0.016" classname="testcases.ApplicationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [B] but found [A]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [B] but found [A]
at testcases.ApplicationTest.forgotPassword(ApplicationTest.java:68)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- forgotPassword -->
</testsuite> <!-- Application Test -->
